paths_cli/compiling/_gendocs/config_handler.py

```diff
@@ -0,0 +1,27 @@
+from collections import namedtuple
+from paths_cli.commands.compile import select_loader
+
+DocCategoryInfo = namedtuple('DocCategoryInfo', ['header', 'description',
+                                                 'type_required'],
+                             defaults=[None, True])
```

Comment:
just making sure that this is intended to be: `DocCategoryInfo(header, description=None, type_required=True)` (so that the 2 defaults make sense)